### ISSUE TITLE
[Support]:  Failed to connect HACS integration

### ISSUE THREAD

**Issue by lockkp1** [blakeblackshear/frigate#2714]
### Describe the problem you are having

Needing some help. I cannot get the Frigate integration to work in Home Assistant. I have read issue #694 which seemed like a really similar situation to mine but didn't have any luck. I have tried installing the v2.20 and v2.21 HACS integration of Frigate but it still wont work. I have also tried uninstalling Frigate Supervisor Add-On completely, then reinstalling that didn't work. I then tried Frigate NVR (Full Access) but still having issues.

Frigate works fine in the Add-On section, its fully functional but cannot get the integration to work. I am running HassOS on an intel nuc

### Version

0.9.4-26AE608

### Frigate config file

```yaml
mqtt:
 host: *** (redacted)
 user: mqttuser
 password: *** (redacted)

 
cameras:
  garage_cam:
    ffmpeg:
      inputs:
        - path: rtsp://admin:****@*****/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
      width: 640
      height: 480
      fps: 5
 
detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
```

### Relevant log output

```shell
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-01-26 19:17:55] frigate.app                    INFO    : Starting Frigate (0.9.4-26ae608)
[2022-01-26 19:17:55] frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
[2022-01-26 19:17:55] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-01-26 19:17:56] peewee_migrate                 INFO    : There is nothing to migrate
[2022-01-26 19:17:56] frigate.mqtt                   INFO    : MQTT connected
[2022-01-26 19:17:56] frigate.app                    INFO    : Output process started: 216
[2022-01-26 19:17:56] ws4py                          INFO    : Using epoll
[2022-01-26 19:17:56] frigate.app                    INFO    : Camera processor started for garage_cam: 221
[2022-01-26 19:17:56] frigate.app                    INFO    : Capture process started for garage_cam: 222
[2022-01-26 19:17:56] detector.cpu1                  INFO    : Starting detection process: 214
[2022-01-26 19:17:56] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-01-26 19:17:56] detector.cpu2                  INFO    : Starting detection process: 215
[2022-01-26 19:17:56] ws4py                          INFO    : Using epoll
[2022-01-26 19:17:56] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-01-26 19:18:14] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45844]
[2022-01-26 19:18:18] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45844]
[2022-01-26 19:23:07] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:46938]
[2022-01-26 19:23:14] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:47902]
[2022-01-26 19:23:23] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:47902]
[2022-01-26 19:23:23] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:46938]
[2022-01-26 19:24:53] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:47224]
[2022-01-26 19:24:54] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:47224]
[2022-01-26 19:30:21] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:48166]
[2022-01-26 19:30:24] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:48166]
[2022-01-26 19:30:30] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:48290]
[2022-01-26 19:30:39] frigate.mqtt                   INFO    : Turning off detection for garage_cam via mqtt
[2022-01-26 19:30:40] frigate.mqtt                   INFO    : Turning on detection for garage_cam via mqtt
[2022-01-26 19:30:47] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:48290]
```

### FFprobe output from your camera

```shell
not sure how to do this
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Dahua DH-P20T1

**Comment by blakeblackshear** [comment]
What is the url you are using when setting up the integration? Also, which specific addon do you have running?

**Comment by lockkp1** [comment]
hi blake thanks so much for your reply. I went back to Frigate NVR (regular non full access) v2.3 running at the moment and then no link it was a straight download from HACS v2.2.1.

I finally managed to get it to work though, but Im not sure if I’ll run into some issues down the road as I am still getting error messages in my home assistant log about connecting.  The MQTT entities show up and they are working but I am afraid it might stop. 

So what I did was to switch to a router with VPN. Things installed smoothly and then miraculously the integration setup smoothly as well. I then shut down my home assistant and went back to the old network without VPN. Things work but as mentioned still getting those connection errors. 

Thank you so much again
